dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.beanValidationCDI-3.1.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.beanValidationCDI-3.1
+visibility=private
+IBM-App-ForceRestart: install, uninstall
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.cdi-3.0)(osgi.identity=io.openliberty.cdi-4.0)(osgi.identity=io.openliberty.cdi-4.1)))", \
```

Comment:
```suggestion
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.cdi-4.1)))", \
```
Since cdi-4.1 is the EE11 version, I don't believe there's value to including 3.0 and 4.0